crates/oxc_semantic/src/reference.rs

```diff
@@ -10,8 +11,10 @@ define_index_type! {
 #[derive(Debug, Clone)]
 pub struct Reference {
     span: Span,
+    /// The name of the identifier that was referred to
     name: Atom,
     symbol_id: Option<SymbolId>,
+    /// Nature of the reference usage
```

Comment:
"Nature" has no meaning in this context, something like "Read or write of the usage"?

Reply:
Doesn't this feel a little redundant? How about something like "How other expressions use the reference"?

Reply:
this is still not clear to me what it would mean lol

Reply:
comment has been changed